index.js

```diff
@@ -262,12 +262,13 @@ console.log(century)
  * Note that sucessfully invoking this function multiple times without refreshing your browser will continuously remove artists from the array until there are none left. If you refresh your browser, the data will reset.  
 */
 function removeArtist(arr, index) {
-    return arr.splice[index];
+    arr.splice(index, index);
+    return arr;
   }
   
-  removeArtist(artists, 0)
+removeArtist(artists, 0)
+console.log(removeArtist(artists, 0))
 
```

Comment:
{/*this is for task 6 addArtist()*/}


{/* feedback
  //break it into steps
  // what things do you have access to variables etc arrays var consts lets
    // the array of artists
    // each artist is an object
  
    //what do you want them to be in the end when you return or console.log
    // you want it to be the same array with one new artist added

  //what steps can you take to get there
    // create a function
    // accept parameters for the values of an artist object
    // create a newArtis object
    // plug the parameters into the key value pairs on the new Artist object
    // take the old array and push the new artist object onto array of artists
    // return the updata array of artists
    // console.log the function running with new arits parameters plugged in as arguments

*/
}